src/Akeneo/Platform/Bundle/ImportExportBundle/Resources/config/requirejs.yml

```
@@ -4,3 +4,5 @@ config:
 
         pim/importexport/template/widget/last-operations-widget: pimimportexport/templates/last-operations-widget.html
         pim/importexport/template/widget/view-all-btn: pimimportexport/templates/view-all-btn.html
+
+        pimimportexport/js/JobExecutionStatus: pimimportexport/js/JobExecutionStatus.tsx
```

```suggestion
        pimimportexport/js/JobExecutionStatus: pimimportexport/js/JobExecutionStatus
```
It should not have the extension, webpack loader is responsible to find the correct extension.
FYI, there is a hack in our current webpack config to allow extensions because otherwise it throw an error.
Ideally, it's good to not add new instances of it :)

---

done